src/Components/Pagination/Pagination.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Pagination from './Pagination';
+import { shallow, mount } from 'enzyme';
+import { shallowToJson } from 'enzyme-to-json';
+
+describe('Pagination HTML renders correctly', () => {
+    it('Pagionation check with correct props', () => {
+        const output = shallow(
+            <Pagination actualPage={1} totalCount={879} changingPage={() => {}}/>
+        )
+        expect(shallowToJson(output)).toMatchSnapshot()
+    })
+
+    it('Pagination goes on next page', () => {
+        const spy = jest.fn();
+        const output = mount(
+            <Pagination actualPage={1} totalCount={879} changingPage={spy}/>
+        )
+        output.find('[name="buttonNext"]').simulate('click');
+        expect(spy.mock.calls.length).toBe(1)
```

Comment:
Please add also argument checking

expect(spy.mock.calls[0][0]).toBe('next')

Reply:
Added for both tests prev/next